app/src/main/java/gis2018/udacity/tametu/utils/Utils.java

```diff
@@ -51,7 +51,8 @@ public static int updateWorkSessionCount(SharedPreferences preferences, Context
      */
     public static int getTypeOfBreak(SharedPreferences preferences, Context context) {
         int currentWorkSessionCount = preferences.getInt(context.getString(R.string.work_session_count_key), 0);
-        if (currentWorkSessionCount % 4 == 0)
+        int session= preferences.getInt(context.getString(R.string.start_long_break_after), 2);
+        if (currentWorkSessionCount%session==0)
```

Comment:
Consider introducing spaces in between. This will make the code look better.
As a general practice, you can use `Ctrl + Alt + L` (Reformat Code) often and this will format the code as per the style of this project.